core/new-gui/src/app/workspace/component/visualization-panel-content/visualization-panel-content.component.ts

```diff
@@ -23,6 +30,7 @@ import { Subscription } from 'rxjs';
 export class VisualizationPanelContentComponent implements AfterViewInit, OnDestroy {
   // this readonly variable must be the same as HTML element ID for visualization
   public static readonly CHART_ID = '#texera-result-chart-content';
+  public static readonly MAP_CONTAINER = 'texera-result-map-container';
```

Comment:
should it match as `MAP_CONTAINER_ID`, also include the hashtag?

Reply:
should not include the hashtag, the library requirement